src/openwhisk/static.js

```diff
@@ -58,7 +58,7 @@ function isBinary(type) {
     return false;
   }
   if (type.match(/.*\/.*xml/)) {
-    return false;
+    return !!type.match(/svg/); // openwshisk treats svg as binary
```

Comment:
I think openwhisk treats all `image/*` as binary. maybe that would be the easier fix.

Reply:
So if I understand you correctly, you mean adding a
```
if (type.match(/image\/.*/)) {
    return true;
}
```
above?

Reply:
or rather wrap all the existing `if`s in a 
```
if (!type.match(/image\/.*/)) {
    ...
}
```

Reply:
we should use the same code as openwhisk. I think it's here:
https://github.com/apache/incubator-openwhisk/blob/master/core/controller/src/main/scala/org/apache/openwhisk/core/controller/WebActions.scala#L325

but never mind for now.